pom.xml

```diff
@@ -223,7 +204,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-aether-provider</artifactId>
-      <version>${mavenVersion}</version>
+      <version>3.3.9</version>
```

Comment:
I think we will quickly have a nightmare to resolve with the 3 versions of aether (Maven 3.5.0 added a new lib) cc @olamy